packages/cozy-harvest-lib/src/components/cards/LaunchTriggerAlert.jsx

```diff
@@ -1,57 +1,83 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 
 import Alert from 'cozy-ui/transpiled/react/Alert'
 import Button from 'cozy-ui/transpiled/react/Buttons'
 import Typography from 'cozy-ui/transpiled/react/Typography'
 import Spinner from 'cozy-ui/transpiled/react/Spinner'
+import Snackbar from 'cozy-ui/transpiled/react/Snackbar'
 
-import KonnectorIcon from '../KonnectorIcon'
 import { getLastSuccessDate, getKonnectorSlug } from '../../helpers/triggers'
 import { isRunnable } from '../../helpers/konnectors'
 import { useFlowState } from '../../models/withConnectionFlow'
+import { SUCCESS } from '../../models/flowEvents'
+import KonnectorIcon from '../KonnectorIcon'
 import { makeLabel } from './helpers'
 
 export const LaunchTriggerAlert = ({ flow, f, t, disabled }) => {
-  const { trigger, running, expectingTriggerLaunch } = useFlowState(flow)
+  const [showSuccessSnackbar, setShowSuccessSnackbar] = useState(false)
+  const { trigger, running, expectingTriggerLaunch, status } =
+    useFlowState(flow)
   const { launch, konnector } = flow
+
   const lastSuccessDate = getLastSuccessDate(trigger)
   const isKonnectorRunnable = isRunnable({ win: window, konnector })
+  const isSuccess = status === SUCCESS
```

Comment:
should be a useCallback no? Since you use this `isSuccess` for the dep of your useEffect in order to se the `setShowSuccess...`. If the component is re-rendered you will `setShowSuccessSnackbar` several times. 

Or if the user has closed the snackbar once, but then this component is re-rendered, then the snackback will be shown again

Reply:
Also, there is already a lot of concept inside Harvest and the Konnector flow. If we can avoid the creation of new indirection or else, it can be good. 

Here, the `isSuccess` can be confusing. What are we talking about? We need to go down the file to read the definition... It seems to me easier to have a : `if(status === SUCCESS){ setShow...`